develop-docs/self-hosted/index.mdx

```diff
@@ -73,6 +73,10 @@ Thirty (30) day retention. No marketing. Privacy policy at sentry.io/privacy.
 
 Starting with the 22.10.0 release, we will require those running the Sentry installer to choose to opt in or out. If you are running the installer under automation, you may want to set `REPORT_SELF_HOSTED_ISSUES` or pass `--(no-)report-self-hosted-issues` to the installer accordingly.
 
+### Air-gapped installation
+
+Sentry can run in air-gapped environments, where there are no internet access. Depending on how air-tight your environment is, the best way to do so is to do installation and upgrades behind a proxy, see [Installing Behind a Proxy](#installing-behind-a-proxy) below. If it's not possible, you may build the entire Docker Compose stack on another machine, use `docker commit` to "freeze" the images, then use combination of `docker save` and `docker load` which convert the image as a `.tar` file and transfer it into the air-gapped machine. Then, you can use `docker-compose up` to start the services. Although this may seem possible, we don't provide any further help for this use case, as it's not a common scenario.
```

Comment:
@aldy505 is it really too hard to write and maintain such a script? I can see that being useful for backups before upgrades: in case something goes wrong, we might just allow people to rollback very quickly (at the expense of disk space?)

Reply:
Well I believe it depends on the person's capability and preference on how to do rollbacks. Some might just create a Virtual Machine snapshot just before upgrade, and restore from that.

Personally, I wouldn't trust docker commit + docker export that much, since on Sentry we have a lots of moving parts other than just the Docker image.

Reply:
Fair enough 🙂